dse/Dockerfile-dse6.8.jdk8

```diff
@@ -19,16 +19,14 @@ ARG DOWNLOAD_URL=${URL_PREFIX}/${TARBALL}
 ARG DSE_AGENT_VERSION=6.8.39
 ARG DSE_AGENT_TARBALL=datastax-agent-${DSE_AGENT_VERSION}.tar.gz
 ARG DSE_AGENT_DOWNLOAD_URL=${URL_PREFIX}/${DSE_AGENT_TARBALL}
-
+ADD ${DOWNLOAD_URL} /${TARBALL}
```

Comment:
Whitespace might not work correctly?

Reply:
I guess when I put the `ADD` line, I just put it in place of an existing blank line. Using `ADD` here seems to allow for better layer cache hits as opposed to using a `RUN` command that downloads the bundle.

Also, for the Management API artifact layers, if we aren't building the code in the image, we can get away with a tiny Alpine image. The extra benefit of `ADD` here is that we don't need a base image that has `curl` or `wget` just to pull the bundles and unzip them. So I use the `ADD` lines there for that benefit as well.

Reply:
I didn't mean the whitespace line, but the whitespace in the ADD command. This will result in incorrect URL with encoded whitespace between ${DOWNLOAD_URL} and the `/`character.